index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>
+    Apple home page
+  </title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="./css/style.css">
+</head>
+
+<body>
+  <main>
+    <nav>
+      <div class="nav">
+        <div class="logo"><img src="./images/logo.png" alt=""></div>
+        <a href="">Mac</a>
+        <a href="">iPod</a>
+        <a href="">iPhone</a>
+        <a href="">iPad</a>
+        <a href="">iTune</a>
+        <a href="">Support</a>
+        <div class="seach">
+          <form>
+            <input type="search">
+          </form>
+        </div>
+      </div>
+    </nav>
+    <div class="write">
+      <a href="#">
+        <img src="./images/your_verse_title_small.png" alt="">
+      </a>
+      <a href="#">
+        <img src="./images/your_verse_subhead_small.png" alt="">
+      </a>
+      <p id="lp">
+        <a href="">wach the "Your verse" film and explore sories</a><img src="./images/play.png" alt=""></p>
+    </div>
+  </main>
+  <footer>
+    <div class="footer">
+      <img src="./images/promo_iphone5s.jpg" alt="">
+      <img src="./images/promo_iphone5c.jpg" alt="">
+      <img src="./images/promo_30_years.jpg" alt="">
+      <img src="./images/promo_supplier_responsibility.jpg" alt="">
```

Comment:
Each of these images should be inside an `<a></a>` tag because they are all links.